lib/wdioUi5-index.js

```diff
@@ -149,16 +150,21 @@ function injectUI5(context) {
                                 // function
 
                                 // filter private methods
-                                if (item.startsWith('_')) {
+                                if (item.startsWith("_")) {
```

Comment:
with `prettier`'s single-qoutes setting, shouldn't this be converted on-save to single qoutes (same for entire file)